corehq/apps/app_manager/models.py

```diff
@@ -5410,14 +5410,14 @@ def rearrange_modules(self, i, j):
             raise RearrangeError()
         self.modules = modules
 
-    def rearrange_forms(self, to_module_id, from_module_id, i, j):
+    def rearrange_forms(self, to_module_uid, from_module_uid, i, j):
```

Comment:
hooray

Reply:
I did also consider switching rearrangement to work by UUID, but figured that'd be a bit too much scope creep and wouldn't really make what I'm working on any easier.  Would be nice to do though - instead of saying "rearrange from position x to position y", say "move module X to immediately after module y".  Then even if someone moved things around after you loaded the page, it'd work properly.